Validation/HGCalValidation/python/hgcalPlots.py

```diff
@@ -0,0 +1,2031 @@
+import os
+import copy
+import collections
+
+import six
+import ROOT
+ROOT.gROOT.SetBatch(True)
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+
+from Validation.RecoTrack.plotting.plotting import Plot, PlotGroup, PlotFolder, Plotter, PlotOnSideGroup
+from Validation.RecoTrack.plotting.html import PlotPurpose
+import Validation.RecoTrack.plotting.plotting as plotting
+import Validation.RecoTrack.plotting.validation as validation
+import Validation.RecoTrack.plotting.html as html
+
+#To be able to spot any issues both in -z and +z a layer id was introduced 
+#that spans from 0 to 103. The mapping is : 
+#-z: 0->51
+#+z: 52->103
+lastLayerEEzm = 28  # last layer of EE -z
```

Comment:
these values will also have to be kept in sync with different geometries

Reply:
Initially, I thought of adding these values as options in the running command. However, makeHGCalValidationPlots.py that could take 
these values already run hgcalPlots.py and I cannot pass them since this is already imported.